app/boards/interconnects/blackpill/blackpill.zmk.yml

```diff
@@ -0,0 +1,10 @@
+file_format: "1"
+id: blackpill
+name: BlackPill
+type: interconnect
+url: https://github.com/WeActStudio/WeActStudio.MiniSTM32F4x1
+manufacturer: WeAct Studio
+description: |
+  The WeAct Studio BlackPill has grown in popularity due to its low price, availability, and utilization of the powerful STM32F4x1CEU6 microcontroller. The BlackPill features more GPIO than most other boards, but also has a comparatively larger footprint as a result. Many clones and variations of the original BlackPill are available on the market as an affordable and more powerful alternative to many popular boards. The official WeAct variations of the WeAct Studio BlackPill are powered by the STM32F411CEU6 and STM32F401CEU6 microcontrollers.
+
+  Note: ZMK does not currently support STM32F4x1CEU6 based BlackPills, but the framework exists in Zephyr to add support in the future as a non-wireless option.
```

Comment:
Not sure I understand what you mean here... There are upstream board definitions for several black pill boards, but they're missing the interconnect bits and ZMK_USB Kconfig.

Is that what you mean is missing?

Reply:
Still waiting on a response to this question.